assets/javascript/app.js

```diff
@@ -1,348 +0,0 @@
-var countAsync = 0;
-var divMatches = $("<div>");
-
-
-function isURL(imageData) {
```

Comment:
Alternatively, you could using an input with the type of url to validate the input.

<form id='urlForm' action="">
  <input type="url">
  <button></button>
</form>

$('#urlForm').on('submit', function (event) {
  event.preventDefault();
  // code to run when user submits a valid url
})